ush/python/pygfs/task/land_analysis.py

```diff
@@ -336,6 +339,15 @@ def finalize(self) -> None:
         statfile = os.path.join(self.task_config.COM_LAND_ANALYSIS, f"{self.task_config.APREFIX}landstat.tgz")
         self.tgz_diags(statfile, self.task_config.DATA)
 
+        logger.info("Copy full YAML to COM")
+        src = os.path.join(self.task_config['DATA'], f"{self.task_config['CDUMP']}.t{self.runtime_config['cyc']:02d}z.letkfoi.yaml")
+        dest = os.path.join(self.task_config.COM_LAND_ANALYSIS, f"{self.task_config['CDUMP']}.t{self.runtime_config['cyc']:02d}z.letkfoi.yaml")
+        yaml_copy = {
+            'mkdir': [self.task_config.COM_LAND_ANALYSIS],
+            'copy': [[src, dest]]
+        }
+        FileHandler(yaml_copy).sync()
```

Comment:
Why? 
`COM` is for `data` not configuration.  The UFS-weather-model has set a precedent for copying namelists and configuration stuff into `COM`.  If there is a need to copy this data for every cycle, we need a proper solution with a well defined set of requirements.

Reply:
Thanks @CoryMartin-NOAA. In addition, I include copying the YAML files for archive to keep consistent with other DA component such as aero DA.

Reply:
An update to this comment:

After discussions w/ NCO, if we wish to retain cycle specific configuration artifacts such as namelists, etc. that are not already part of `HOMEgfs`, we can do so in a single location in `COM`.  @CoryMartin-NOAA and @aerorahul discussed a good place for said configuration artifacts would be `ROTDIR/RUN.PDY/cyc/conf`.
The jobs will need to ensure that they identify the artifacts uniquely; e.g. for e.g. `input.nml` from the ufs-weather-model should be save as `ufs_input.nml` 

This will require creating a `COM_CONF_TMPL` in `config.com` and then filling the template with appropriate substitutions in the j-jobs that wish to save the artifacts.

Please let me know if you have any questions.
@WalterKolczynski-NOAA FYI

Reply:
Thanks @aerorahul for the updates. 
@CoryMartin-NOAA @aerorahul Do you want me to include this change in this PR by putting the yaml file into the `ROTDIR/RUN.PDY/cyc/conf`?

Reply:
As suggested, I create `COM_CONF_TMPL` in `config.com` and copy the YAML file into `ROTDIR/RUN.PDY/cyc/conf`. I still keep using the previous name (e.g., gdas.t18z.letkfoi.yaml). What do you suggest the change for the file name?